### ISSUE TITLE
训练效果差

### ISSUE THREAD

**Issue by mazx21** [hellen9527/Pytorch-Chinese-Text-Classification#4]
![QQ图片20240131104840](https://github.com/hellen9527/Pytorch-Chinese-Text-Classification/assets/105896608/fd98fb02-f676-4320-8ba9-773beaa22031)
您好 我想问一下换了自己的数据集为什么效果这么差呢？如何提升啊

**Comment by yuye2133** [comment]
看起来label=2的数据没怎么被训练到，确认下训练数据里有多少label=2的数据，如果太少，试下focal_loss或者加大label=2的loss权重。另外run_bert_cls.sh默认是直接做测试的，没有训练，要把do_test改成do_train；我这边直接把示例数据里包含“一般”，“普通”，“还行”之类的词作为第三个label，看训练过程
Iter:    100,  Train Loss: 0.057,  Train Acc: 84.64%,  Val Loss:  0.19,  Val Acc: 95.35%
Iter:    200,  Train Loss: 0.075,  Train Acc: 95.97%,  Val Loss:  0.11,  Val Acc: 96.68%
Iter:    300,  Train Loss: 0.088,  Train Acc: 97.94%,  Val Loss: 0.062,  Val Acc: 98.28%
Iter:    400,  Train Loss: 0.0053,  Train Acc: 98.19%,  Val Loss: 0.069,  Val Acc: 98.44%
loss在正常下降，说明在收敛，整个过程是正常的，你可以贴一下你的loss变化

**Comment by mazx21** [comment]
![image](https://github.com/hellen9527/Pytorch-Chinese-Text-Classification/assets/105896608/e6c8cb58-673b-4dae-a201-c3468c515511)
请问这是什么原因呢
